vagrant/ansible/roles/client.prep/tasks/main.yml

```diff
@@ -17,7 +17,9 @@
 
 - name: Install Python yaml package
```

Comment:
The `name` is not entirely correct anymore? :smiley:

Reply:
pip on RHEL 7 installs for python 2. So the iso8601 install installs the library for python 2. This is a problem with selftest scripts as they use python 3 and we need the library installed for Python 3.
Therefore for RHEL 7, We have an exception to install the rpm package in the previous commit.

Using pip shouldn't be a problem in newer distros since they should pick python 3 by default.

regd: name
Sorry. That change got lost when I had to sort out a conflict when doing a rebase.

Reply:
The name for the ansible action when installing the python module using pip has been resolved in the next commit.